package.sh

```diff
@@ -157,16 +163,25 @@ else
 	##    - building binaries from sources
 	##    - copy artifacts to the build_dir
 	echo "Build the module..."
-	if [ -e ./packaging/build.sh ]; then
-		./packaging/build.sh $PWD ${build_dir} || exit 1
-	else
+	if [ ! -e ./packaging/build.sh ]; then
 		echo "ERROR: No build script available"
 		exit 1
 	fi
+	if [ "${kernel_config}" == "" ]; then
+		./packaging/build.sh $PWD ${build_dir} || exit 1
+	else
+		### Kernel build needs to know which configuration to be built.
+		./packaging/build.sh $PWD ${kernel_config} || exit 1
```

Comment:
Could this be done with one call, so that the config  would be the third parameter?
./packaging/build.sh $PWD ${package_dir} ${kernel_config}
(this example assumes the "build_dir" is changed to better describing name "package_dir")